bridges/NordbayernBridge.php

```diff
@@ -119,7 +119,12 @@ private function handleArticle($link) {
 	private function handleNewsblock($listSite) {
 		$main = $listSite->find('main', 0);
 		foreach($main->find('article') as $article) {
-			self::handleArticle(self::URI . $article->find('a', 0)->href);
+			$aLink = $article->find('a', 0)->href;
+			if(substr($aLink, 0, 4) === 'http') {
+				self::handleArticle($aLink);
+			} else {
+				self::handleArticle(self::URI . $aLink);
+			}
```

Comment:
I suggest to use `urljoin($url_region, $article->find('a', 0)->href)`. First param is url of region page. AFAIK it is `self::URI . '/region/' . $region` from `collectData`